data/json/npcs/talk_tags.json

```diff
@@ -565,7 +583,122 @@
           "Look out!  A",
           "<swear!>, here comes a <swear>"
         ]
-    },{
+    },
+    {
+      "type": "snippet",
+      "category": "<general_danger>",
+      "text": [
+        "Look sharp!  Things are heating up.",
+        "Hey, <name_g>!  Hostiles inbound.",
+        "Uh, <name_g>?  Are we fighting or leaving?",
+        "Naptime is over.",
+        "Who's there?",
+        "Hello?",
+        "Look alive!"
+      ]
+    },
+    {
+      "type": "snippet",
+      "category": "<general_danger_h>",
+      "//": "Some lines deliberately copied from mi-go text",
```

Comment:
👍

Reply:
Should the lines common to NPCs and Mi-gos be factored out into their own snippet so they need not be duplicated and will remain in sync?

Reply:
So the npc talk_tags snippets in data/json/npcs/talk_tags.json are actually in a fairly different format than migo speech in data/json/speech.json, because speech.json is actual sounds with volumes and talk_tags are just snippets that get merged into other text.  So even if there's value in having them stay in sync - and mi-gos say more than goons do and vice versa - it's beyond the scope of this PR.

Having mi-gos pull stuff from NPC speech would also be cool, but mi-gos repeat a lot of stuff that is clearly pre-Cataclysm and we'd want to preserve that.  And again, beyond the scope.